tensorrt_llm/_torch/auto_deploy/transform/library/_chat_template.py

```
@@ -0,0 +1,443 @@
+
+    Args:
+        chat_template_path: Path to the chat template JSON file
+
+    Raises:
+        ValueError: If the chat template is invalid
+        FileNotFoundError: If the chat template file doesn't exist
+    """
+    print(f"Validating chat template: {chat_template_path}")
+
+    if not os.path.exists(chat_template_path):
+        raise FileNotFoundError(f"Chat template file not found: {chat_template_path}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)